### ISSUE TITLE
Improve the Setup Screens UI

### ISSUE THREAD

**Issue by latenitefilms** [fcpxhacks/fcpxhacks#2061]
This looks WAY nicer:

<img width="952" alt="Screen Shot 2019-10-18 at 11 36 13 pm" src="https://user-images.githubusercontent.com/22286696/67246863-62760f00-f4ab-11e9-861f-1499e16e43ac.png">
<img width="952" alt="Screen Shot 2019-10-18 at 11 00 35 pm" src="https://user-images.githubusercontent.com/22286696/67247532-699e1c80-f4ad-11e9-8636-7ca41ab68584.png">
<img width="952" alt="Screen Shot 2019-10-18 at 11 18 24 pm" src="https://user-images.githubusercontent.com/22286696/67247533-699e1c80-f4ad-11e9-916e-2e413d6cf636.png">

**Comment by latenitefilms** [comment]
This is also really nice:

<img width="1020" alt="Screen Shot 2019-11-08 at 4 46 07 pm" src="https://user-images.githubusercontent.com/22286696/68553850-9de77600-0478-11ea-9059-d899d20d5c54.png">

**Comment by randomeizer** [comment]
Yeah, it would be nice to have an automatic "What's New" summary with updates that will only shows once and records what versions have been seen

**Comment by latenitefilms** [comment]
This is quite a nice Sparkle layout too:

<img width="972" alt="Screen Shot 2019-11-13 at 11 45 55 am" src="https://user-images.githubusercontent.com/22286696/68728588-d2476780-061b-11ea-927c-63c93892425b.png">

**Comment by latenitefilms** [comment]
[This](https://www.iospirit.com/blog/2019/11/Remote-Buddy-20-Public-Beta-Available) looks really nice too:

![remotebuddy-2-privacy@2x~08b03eb244710eb0997be4c8c8aa6a81db8f1d06](https://user-images.githubusercontent.com/22286696/69388920-951b5d80-0d1e-11ea-84d1-5cf806eba797.gif)

**Comment by latenitefilms** [comment]
Remote Buddy has really nailed Accessibility Permissions:

<img width="752" alt="Screen Shot 2019-12-26 at 11 54 20 am" src="https://user-images.githubusercontent.com/22286696/71452190-8c62ef00-27d6-11ea-8155-8ef84ee0a887.png">
<img width="899" alt="Screen Shot 2019-12-26 at 11 54 13 am" src="https://user-images.githubusercontent.com/22286696/71452191-8c62ef00-27d6-11ea-8f0d-6fe4922727d0.png">
<img width="962" alt="Screen Shot 2019-12-26 at 11 53 15 am" src="https://user-images.githubusercontent.com/22286696/71452192-8cfb8580-27d6-11ea-8107-69dd48b371ef.png">
<img width="752" alt="Screen Shot 2019-12-26 at 11 52 53 am" src="https://user-images.githubusercontent.com/22286696/71452193-8cfb8580-27d6-11ea-9509-b2ae7fa848ff.png">
<img width="810" alt="Screen Shot 2019-12-26 at 11 52 15 am" src="https://user-images.githubusercontent.com/22286696/71452194-8cfb8580-27d6-11ea-99cc-0bf22ef83249.png">
